workflows/genetic-demux/map-sc-star.nf

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+STARCONTAINER = 'quay.io/biocontainers/star:2.7.9a--h9ee0642_0'
+SAMTOOLSCONTAINER = 'quay.io/biocontainers/samtools:1.14--hb421002_0'
+
+// parameters
+params.ref  = 'Homo_sapiens.GRCh38.104'
+params.star_index = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/star_index/Homo_sapiens.GRCh38.104.star_idx'
+params.barcode_dir = 's3://nextflow-ccdl-data/reference/10X/barcodes' 
+
+params.run_metafile = 's3://ccdl-scpca-data/sample_info/scpca-library-metadata.tsv'
+params.run_ids = 'SCPCR000533'
+
+params.outdir = 's3://nextflow-ccdl-results/scpca/starsolo/'
+
+// 10X barcode files
+cell_barcodes = ['10Xv2': '737K-august-2016.txt',
+                 '10Xv3': '3M-february-2018.txt',
+                 '10Xv3.1': '3M-february-2018.txt',
+                 '10Xv2_5prime': '737K-august-2016.txt']
+
+// supported single cell technologies
+single_cell_techs = cell_barcodes.keySet()
+
+process starsolo{
+  container STARCONTAINER
+  label 'bigdisk'
+  memory "32.GB"
+  cpus "8"
+  input:
+    tuple val(meta), path(read1), path(read2)
+    path star_index
+    path barcode_file
+  output:
+    tuple val(meta), path(output_dir), emit: starsolo_dir
```

Comment:
If you aren't using the first output, do you need to also output the metadata here since you are outputting it with the bam file in the second output? 
```suggestion
    path(output_dir), emit: starsolo_dir
```

Reply:
I actually want to keep this, with the thought that if we did want to proceed with the starsolo results into a different channel, we would still want the next step in the workflow to have the metadata. The cost of outputting it is tiny.